### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by andyk-ms**
Update readme, remove references to preview.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,13 +1,5 @@
 # Microsoft Azure IoT SDK for .NET
 
-## V2 clients release notice
```
Comment (rido-min) [inline readme.md:3]: wondering if we should be more explicit to explain what are the recent plan changes wrt V2 SDK.

**Review by ryanwinter — APPROVED** [review]

**Review by timtay-microsoft — APPROVED** [review]

**Review by lt72 — APPROVED** [review]

**Review by varunpuranik — APPROVED** [review]

**Pull request merged**